news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md

```diff
@@ -0,0 +1,189 @@
+
+
+### Docker images
+* [#239](https://github.com/PrestaShop/docker/pull/239): Fix outdated license headers, by [@matks](https://github.com/matks)
+* [#233](https://github.com/PrestaShop/docker/pull/233): Refactor how images are built, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### PrestaShop Specifications
+* [#178](https://github.com/PrestaShop/prestashop-specs/pull/178): Update Order Page View.md. Thank you [@MatShir](https://github.com/MatShir)
+* [#177](https://github.com/PrestaShop/prestashop-specs/pull/177): Add link for product images in product listing, by [@marionf](https://github.com/marionf)
+* [#175](https://github.com/PrestaShop/prestashop-specs/pull/175): Add recycled packaging information, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#174](https://github.com/PrestaShop/prestashop-specs/pull/174): Add specs for round on each item, by [@marionf](https://github.com/marionf)
+* [#172](https://github.com/PrestaShop/prestashop-specs/pull/172): Edit Shop Parameters > Search, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+
+
+### Auto Upgrade module
+* [#352](https://github.com/PrestaShop/autoupgrade/pull/352): Remove VSCode config files, by [@matks](https://github.com/matks)
+* [#351](https://github.com/PrestaShop/autoupgrade/pull/351): Add travis badge status, by [@matks](https://github.com/matks)
+
+
+### Webservices PHP Client
+* [#73](https://github.com/PrestaShop/PrestaShop-webservice-lib/pull/73): Remove duplicate @throws in PHPDoc. Thank you [@davidglezz](https://github.com/davidglezz)
+
+
+### Docker internal images
+* [#34](https://github.com/PrestaShop/docker-internal-images/pull/34): Use 1.7 instead of 1.7-7.2-apache, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Prestashop UI Kit
+* [#119](https://github.com/PrestaShop/prestashop-ui-kit/pull/119): Update UIKit version inside the readme, by [@NeOMakinG](https://github.com/NeOMakinG)
+
+
+### Product Comments module
+* [#81](https://github.com/PrestaShop/productcomments/pull/81): Release 4.2.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#80](https://github.com/PrestaShop/productcomments/pull/80): Bump to 4.2.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### PHP Developer Tools
+* [#39](https://github.com/PrestaShop/php-dev-tools/pull/39): phpstan config : add  `_PS_MODE_DEV_` to dynamicConstantNames. Thank you [@SebSept](https://github.com/SebSept)
+
+
+### Changes in developer documentation
+* [#801](https://github.com/PrestaShop/docs/pull/801): More informations for maintainers : red flags for code reviews, BC breaks, what it means to approve a PR, by [@matks](https://github.com/matks)
+* [#782](https://github.com/PrestaShop/docs/pull/782): Add tinymce config extend documentation, by [@NeOMakinG](https://github.com/NeOMakinG)
+
+
+## Where to start contributing?
```

Comment:
@LouiseBonnard 🎁

Reply:
😍

Reply:
So nice, thanks a million! Let me know if you need a hand while waiting for it to be automatized, I can pick one every week and write a small introduction. :-)